backend/src/app.service.ts

```diff
@@ -0,0 +1,82 @@
+import { Injectable } from '@nestjs/common';
+import { TodoDto } from './dto/todoDto';
+import { ConnectionService } from './app.connectionService';
+
+@Injectable()
+export class AppService {
+  private connectionService = new ConnectionService();
+
+  async create(todo: TodoDto): Promise<any> {
+    const newTodoOrder = await this.getNextTodoMaxOrder();
+    const insertTodoQuery = `
+    INSERT INTO "todosTable" (id,title,date,status,todosorder)
+    VALUES ($1,$2,$3,$4,$5)
+    RETURNING *;`;
```

Comment:
usually we add another interface for the queries, so the service doesn't interact with the db directly, e.g. `TodosDatabase` class